libraries/AP_Logger/LogStructure.h

```diff
@@ -1212,6 +1210,11 @@ struct PACKED log_Arm_Disarm {
 #define ARSP_UNITS "snPOPP----"
 #define ARSP_MULTS "F00B00----"
 
+#define RFND_LABELS "TimeUS,instance,Dist,Stat,Orient"
```

Comment:
There's no longer any need to break these out into #defines.  Leaving them inline means you don't need to go hunting for them.

Please move these back in-line.

Reply:
fixed